.github/workflows/wheels-manylinux-test.yml

```diff
@@ -0,0 +1,165 @@
+name: Build RAPIDS manylinux wheels
+
+on:
+  workflow_call:
+    inputs:
+      # repo and branch
+      repo:
+        type: string
+      branch:
+        type: string
+      date:
+        type: string
+      sha:
+        type: string
+      build_type:
+        required: true
+        type: string
+
+      # general settings
+      package-name:
+        required: true
+        type: string
+
+      # test settings
+      test-docker-options:
+        required: false
+        type: string
+        default: '-e _NOOP'
```

Comment:
I don't recall exactly how ucx-py uses this option, I assume something breaks if the default here is removed altogether.

Reply:
Yes, we have to use the `options:` key of the `container:` block of GHA (line #99 of this file), and `-e _NOOP` just passes a dummy env var called _NOOP to have a valid string. Empty strings lead to a `docker ... <args> ''` which is invalid.